### DISCUSSION TITLE
Sending mail issue

### DISCUSSION THREAD

**Discussion by Div-shri** [postalserver/postal#2703] [Help with using Postal]
Dear Team,

I have done proper configuration but still while sending the mails from xyz ID but the receipents reeive the mails from abc ID??
How it overcome? Kindly assist.

Where I need to have the configuration setting properly?

**Comment by willpower232** [comment]
You mean you're sending emails with one email address and it looks different when the recipient receives it? You might need to look at the sender header options.
